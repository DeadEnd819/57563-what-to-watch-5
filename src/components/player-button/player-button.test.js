import React from "react";
import renderer from "react-test-renderer";
import PlayerButton from "./player-button";

const noop = ()=>{};

describe(`Render PlayerButton`, () => {
  it(`Should PlayerButton render correctly`, () => {
    const tree = renderer
      .create(
          <PlayerButton isPlaying={true} onTogglePlayButtonClick={noop}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should Header render correctly`, () => {
    const tree = renderer
      .create(
          <PlayerButton isPlaying={false} onTogglePlayButtonClick={noop}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
