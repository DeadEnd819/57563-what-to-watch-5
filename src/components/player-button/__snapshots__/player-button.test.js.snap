// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlayerButton Should Header render correctly 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;

exports[`Render PlayerButton Should PlayerButton render correctly 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlinkHref="#pause"
    />
  </svg>
  <span>
    Pause
  </span>
</button>
`;
